internal/planner/plansdk/plansdk.go

```diff
@@ -91,11 +93,34 @@ type NixpkgsInfo struct {
 	Sha256 string
 }
 
-// The commit hash for nixos-22.11 on 2022-12-06 from status.nixos.org
-const DefaultNixpkgsCommit = "52e3e80afff4b16ccb7c52e9f0f5220552f03d04"
+// The commit hash for nixpkgs-unstable on 2023-01-25 from status.nixos.org
```

Comment:
not convinced about changing from nixpkgs-stable to unstable....started an internal discussion in slack.

Reply:
Commented in Slack.